.github/workflows/build.yml

```diff
@@ -394,7 +394,7 @@ jobs:
             cd python/osrd_schemas
             poetry install --no-interaction --no-root
             poetry run python -m osrd_schemas.infra_editor > current_infra_schema.json
-            diff current_infra_schema.json ../../front/src/reducers/osrdconf/infra_schema.json
+            diff current_infra_schema.json ../../front/src/store/reducers/osrdconf/infra_schema.json
```

Comment:
I'm not sure what this move has to do with this PR, but in any case, you should update the `.github/CODEOWNERS` file too.

Reply:
This change has nothing to do with the subject of the PR, but at first it was supposed to be a small reorganization of the tree structure in front side. However, when applying the change in `.github/CODEOWNERS`, the ci remains broken...

Reply:
Yeah, that wasn't what broke the CI.

The problem is that you have too many warnings during the front build. You should fix those.